building/trusted_applications.rst

```diff
@@ -299,3 +305,66 @@ sequence:
             return TEE_ERROR_NOT_SUPPORTED;
         }
     }
+
+.. _TASign:
+
+Signing of TAs
+**************
+
+All :ref:`REE Filesystem Trusted Applications<ree_fs_ta>` need to be signed. The signature is verified by :ref:`optee_os` upon loading of the TA. Within the :ref:`optee_os` source is a directory ``keys``. The public part of ``keys/default_ta.pem`` will be compiled into the :ref:`optee_os` binary and the signature of each TA will be verified against this key upon loading.
+Currently ``keys/default_ta.pem`` must contain an RSA key.
+
+.. warning::
+
+   :ref:`optee_os` comes with a default **private** key in its source to facilitate easy development, testing, debugging and QA.
```

Comment:
This is a bit of repetition of the first sentences in this document. But, maybe it's better to state it twice than having people doing mistakes.